test/test_helper.rb

```diff
@@ -14,16 +12,14 @@
 
 require "active_storage"
 
-require "active_storage/service"
 require "yaml"
 SERVICE_CONFIGURATIONS = begin
-  YAML.load_file(File.expand_path("../service/configurations.yml", __FILE__)).deep_symbolize_keys
+  YAML.load_file(File.expand_path("../dummy/config/storage_services.yml", __FILE__)).deep_symbolize_keys
```

Comment:
I've moved the configurations inside the dummy, otherwise dummy app will fail on the boot